docs/blender_docs/io_gltf2.rst

```diff
@@ -30,7 +30,198 @@ This importer/exporter supports the following glTF 2.0 features:
+copy the strength setting into the glTF.
+
+.. figure:: /images/addons_io-gltf2-material-normal.png
+   :alt: A normal map image connected such that the exporter will find it and copy it
+         to the glTF file.
+
+   A normal map image connected such that the exporter will find it and copy it
+   to the glTF file.
+
+.. note::
+
+   Blender's "Cycles" rendering engine has a "Bake" panel that can be used to bake
+   tangent-space normal maps from almost any other arrangement of normal vector
+   nodes.  Switch the "Bake type" to "Normal".  Keep the default space settings
+   (Space: Tangent, R: +X, G: +Y, B: +Z) when using this bake panel for glTF.
+   The resulting baked image can be saved and hooked up to a new material using
+   the Normal Map node as described above.
+
+Emissive
+^^^^^^^^
+
+An "Image Texture" node can be connected to an "Emission" shader node, and optionally
+combined with settings from a "Principled BSDF" node by way of an "Add" shader node.
+
+If the glTF exporter finds an image connected to the Emission shader node, it will
+export that image as the glTF material's emissive image.
+
+Double Sided
+^^^^^^^^^^^^
+
+In glTF, double-sided is a property that is applied per-material, not per-viewport
+or per-mesh, so it has no exact equivalent within Blender.  It can be thought of as
+a combination of backface culling (in Blender's viewport) and double-sided lighting
+(a Blender mesh property).
+
+When ``false`` (the default), backface culling is used, and the backs of faces in
+the glTF will not be visible in other software.  When ``true``, backface culling
+is disabled, and double-sided lighting is used, automatically reversing the normal
+vectors of any visible back faces.
+
+To set this value to true, create a custom node group by the name of
+``glTF Metallic Roughness``, add an input value named ``DoubleSided`` with a range
+of 0.0 to 1.0, and set it to 1.0.  There will be no equivalent effect in Blender,
+but the exporter will enable double-sided mode in glTF for this material.
+
+Blend Modes
+^^^^^^^^^^^
+
+The Base Color input value, or Base Color image, can optionally supply alpha values.
+How these values are treated by glTF depends on the selected blend mode.
+
+With the "Eevee" rendering engine selected, each material has a "Blend Mode" on the
+material settings panel.  Use this setting to govern how alpha values from the
+Base Color channel are treated in glTF.  Three settings are supported by glTF:
+
+- **Opaque** - Alpha values are ignored (the default).
+- **Alpha Blend** - Lower alpha values cause blending with background objects.
+- **Alpha Clip** - Alpha values below the ``Clip Threshold`` setting will cause portions
+of the material to not be rendered at all.  Everything else is rendered as opaque.
+
+.. note::
+
+   Be careful using materials with blend mode: **Alpha Blend**.  Some real-time rendering
+   engines may show artifacts if multiple blended polygons overlap.
```

Comment:
I think `Some real-time rendering engines may show artifacts if multiple blended polygons overlap` is somewhat understating the complexity here 😅 – suggested:

>Be aware that transparency (or **Alpha Blend** mode) is complex for real-time engines to render, and may behave in unexpected ways after export. Where possible, use **Alpha Clip** mode instead.